### DISCUSSION TITLE
I want horizontal scrollbars to be more obvious

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#170] [Wants]
On some websites, it’s difficult to know when you are viewing a horizontally-scrollable element if the horizontal scrollbar appears below the bottom edge of the viewport. It would be nice if browsers could affix the scrollbar to the bottom edge of the viewport while the scrollable element’s own bottom edge is out of view. It could snap to the bottom of the element when that bottom edge is revealed (sort of like `position: sticky`, but in reverse).

<hr>

https://webwewant.fyi/wants/109/

**Comment by nikunjbhatt** [comment]
Nice. A CSS property may be defined to handle this like `scrollbar-x: element-bottom | element-top | viewport-bottom | viewport-top | element-both | viewport-both`

The `element-` prefix in the property value would decide the alignment or position or relativity of the scrollbar. The `-top` suffix would show scrollbar at top instead of at the default bottom, and `-both` suffix would show scrollbar at both top and bottom. So, presently `element-bottom` is default.
